src/components/Map/MapToolbar.js

```diff
@@ -1,67 +1,51 @@
 import './MapToolbar.less';
 import { clearStateFilter, showStateComplaints } from '../../actions/map';
-import { coalesce } from '../../utils';
-import { connect } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import iconMap from '../iconMap';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { THESE_UNITED_STATES } from '../../constants';
+import { selectQueryStateAggs } from '../../reducers/query/selectors';
 
-export class MapToolbar extends React.Component {
-  render() {
-    const filteredStates = this.props.filteredStates;
-    return (
-      <div className="map-toolbar">
-        <section className="state-heading">
-          {!filteredStates && <span>United States of America</span>}
-          <span>{filteredStates}</span>
-          {filteredStates && (
-            <a className="clear" onClick={() => this.props.clearStates()}>
-              {iconMap.getIcon('delete-round')}
-              Clear
-            </a>
-          )}
-        </section>
-        {filteredStates && (
-          <section className="state-navigation">
-            <a
-              href="#"
-              className="list"
-              onClick={() => this.props.showComplaints()}
-            >
-              View complaints for filtered states
-            </a>
-          </section>
-        )}
-      </div>
-    );
-  }
-}
-
-export const mapStateToProps = (state) => {
-  const abbrs = coalesce(state.query, 'state', []);
-
-  return {
-    filteredStates: abbrs
-      .filter((x) => x in THESE_UNITED_STATES)
-      .map((x) => THESE_UNITED_STATES[x])
-      .join(', '),
-  };
-};
-
-export const mapDispatchToProps = (dispatch) => ({
-  clearStates: () => {
-    dispatch(clearStateFilter());
-  },
-  showComplaints: () => {
-    dispatch(showStateComplaints());
-  },
-});
+export const MapToolbar = () => {
+  const dispatch = useDispatch();
+  const abbrs = useSelector(selectQueryStateAggs);
```

Comment:
I know this was pulled from earlier code, but could you rename this variable? I'm not sure what it's supposed to be referring to. Abbreviations?